tests/core/test_core.py

```diff
@@ -467,6 +467,50 @@ def test_stream_muting_confirmation_popup(
         )
         pop_up.assert_called_once_with(controller, text(), partial())
 
+    @pytest.mark.parametrize(
+        "narrow, search_query, expected_search_text",
+        (
+            ([["stream", "PTEST"]], "FOO", "FOO stream:PTEST"),
+            (
+                [["stream", "PTEST PTEST2"]],
+                "FOO",
+                "FOO stream:PTEST+PTEST2",
+            ),
+            (
+                [["stream", "PTEST"], ["topic", "Test"]],
+                "FOO",
+                "FOO stream:PTEST topic:Test",
+            ),
+            (
+                [["is", "private"]],
+                "FOO",
+                "FOO is:private",
+            ),
+            (
+                [["is", "mentioned"]],
+                "FOO",
+                "FOO is:mentioned",
+            ),
+            (
+                [["pm_with", "some1@email.com", "some2@email.com"]],
```

Comment:
Is this the correct format? My memory is that each sublist is 2 elements long.

Reply:
Usually it is, but in case of `pm_with`, I have always observed the length of the sublist to be 1 + no. of recipients.

Reply:
See https://github.com/zulip/zulip-terminal/blob/main/zulipterminal/model.py#L288, for example.

Reply:
Yes, I got it now. I actually didn't realise that all the users under `pm_with` are mentioned under the same string.